app/controllers/api/advocates/claims_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Api
+  module Advocates
+    class ClaimsController < ::Advocates::ClaimsController
+      respond_to :json
+
+      def create
+        @claim = Claim.new(claim_params.merge(advocate_id: current_user.id))
+
+        if @claim.save
+          @claim.submit!
+          render json: @claim, message: 'Claim successfully submitted', status: :created
+        else
+          render json: { errors: @claim.errors.full_messages }, status: :unprocessable_entity
+        end
+      end
+
+      def update
+        if @claim.update(claim_params)
+          render json: @claim, message: 'Claim sucessfully updated'
+        else
+          render json: { errors: @claim.errors.full_messages }, status: :unprocessable_entity
+        end
+      end
+
+      def destroy
+        @claim.destroy
```

Comment:
you might want to call @claim.destroyed? to check if it was successful - I might be overly cautious ?